skll/learner/__init__.py

```diff
@@ -1658,7 +1667,25 @@ def learning_curve(self,
         num_examples : list of int
             The numbers of training examples used to generate
             the curve
+
+        Raises
+        ------
+        ValueError
+            If the number of training `examples` is less than 500
         """
+
+        # check that the number of training examples is more than the minimum
+        # needed for generating a reliable learning curve
+        if len(examples) < 500:
+            if not override_minimum:
+                raise ValueError('Number of training examples provided - {} - '
+                                 'is less than the minimum needed - {} - '
+                                 'for the learning curve to be reliable.'.format(len(examples), 500))
```

Comment:
```suggestion
                raise ValueError(f"Number of training examples provided({len(examples)}) "
                                 f"is less than the minimum needed(500) "
                                 f"for the learning curve to be reliable.")
```

Since SKLL is Python 3.6+ compatible, it's always better to use the modern f-strings.